README.md

```diff
@@ -350,6 +350,36 @@ Future.reject('it broke')
 //> Left('it broke')
 ```
 
+#### finally
+##### `#finally :: Future a b ~> Future a c -> Future a b`
+##### `.finally :: Future a b -> Future a c -> Future a c`
```

Comment:
Perhaps it'd be better to leave the output type the same `Future a b` and switch `b` and `c` between the first two types.